classes/classes.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Feb  6 19:30:26 2022
+
+@author: dmitriipodgalo
+"""
+
+
+from Bio.Seq import Seq
+from Bio import SeqIO
+import matplotlib.pyplot as plt
+import seaborn as sns
+import pathlib
+
+
+class Game():
+
+    muscle = 8
+    brain = 5
+    is_pet = False
+
+    def __init__(self, name='Knight', age=21):
+        self.name = name
+        self.coef_age = age ** 0.5
+
+    def gym(self):
+        self.muscle += (0.01 * (100 - self.coef_age))
+
+    def read(self):
+        self.brain += (0.005 * ((100 - self.coef_age)))
+
+    def watch_TV(self):
+        self.brain -= (0.01 * self.coef_age)
+
+    def lay(self):
+        self.muscle -= (0.01 * self.coef_age)
+        self.brain -= (0.005 * self.coef_age)
+
+    def pet(self):
+        if self.is_pet:
+            print(self.name + ', this good for you')
+        elif self.brain + self.muscle > 50:
+            self.is_pet = True
+            print(self.name + ', congrans, now you have a pet')
+        else:
+            print(self.name + ', sorry...')
+
+
+class RNA():
+
+    def __init__(self, rna):
+        self.rna = Seq(rna)
+
+    def translation(self):
+        return str(self.rna.translate())
+
+    def back_transcription(self):
+        return str(self.rna.back_transcribe())
+
+
+class MySet(set):
+
+    def __init__(self, set_):
+        if sum(i < 0 for i in set_):
+            raise ValueError('Have negative item')
+
+        super().__init__(set_)
+
+    def add(self, item):
+        if item > 0:
+            super(MySet, self).add(item)
```

Comment:
Так делали во вторм питоне, в третьем эта запись немного избыточна
```suggestion
            super().add(item)
```